From 7208f5d6c582ed34d756e647123d6200bcffb78a Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Thu, 31 Mar 2016 19:00:08 +0800
Subject: [PATCH] ARM64: rockchip_cros_defcofnig: turn on the chrome platform

This config should be opened since the config used for chromeos.

Change-Id: I52dd22b1c1a707e6d27311337a5be6f0041cb7f9
Signed-off-by: Caesar Wang <wxt@rock-chips.com>
---
 arch/arm64/configs/rockchip_cros_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rockchip_cros_defconfig b/arch/arm64/configs/rockchip_cros_defconfig
index 97be8a491a16..1f2e08b5da31 100644
--- a/arch/arm64/configs/rockchip_cros_defconfig
+++ b/arch/arm64/configs/rockchip_cros_defconfig
@@ -517,6 +517,7 @@ CONFIG_SW_SYNC=y
 CONFIG_SW_SYNC_USER=y
 CONFIG_ION=y
 CONFIG_ION_ROCKCHIP=y
+CONFIG_CHROME_PLATFORMS=y
 CONFIG_COMMON_CLK_RK808=y
 # CONFIG_COMMON_CLK_XGENE is not set
 CONFIG_MAILBOX=y
-- 
2.35.3

